docs/faq/README.md

```diff
@@ -1,3 +1,7 @@
+---
+permalink: 'faq/index.html'
```

Comment:
This is the main thing that I think will really separate this from the Vuepress approach. I've not seen any documentation that would allow for a similar centralized sitemap, so each page that we don't want to resolve to the default (e.g. this file would be `/faq/README/index.html` normally) we would need to ensure had a `permalink` value set. Lots of growth opportunities from there, however. Things like custom page meta data, dynamic navigation creation by collection tagging, etc.